addOns/spider/spider.gradle.kts

```diff
@@ -73,6 +73,8 @@ dependencies {
     compileOnly(parent!!.childProjects.get("formhandler")!!)
     compileOnly(parent!!.childProjects.get("network")!!)
 
+    implementation("io.kaitai:kaitai-struct-runtime:0.10")
```

Comment:
What's the size? (Just curiosity, I can check if you don't know.)

Reply:
21kb
 https://mvnrepository.com/artifact/io.kaitai/kaitai-struct-runtime/0.10